torch/autograd/gradcheck.py

```diff
@@ -102,76 +114,148 @@ def get_stride(size):
             yield x_tensor, x_idx, d_idx
 
 
-def compute_gradient(fn, inputs, x, idx, delta, eps, is_mkldnn):
+def _get_numerical_jacobian(fn, inputs, outputs=None, target=None, eps=1e-3,
```

Comment:
We should probably prefix all other helper functions in this file with `_` so that no one tries to rely on them. An alternative could be to add them to some `torch.autograd._gradcheck` file that has an underscore prefix.

Reply:
Created an issue for this here https://github.com/pytorch/pytorch/issues/55714. It would take many times the effort to have to modify them names here due to all the rebase conflicts.

Reply:
Yeah, as long as we can do this before 1.9 it doesn't matter when it happens